release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -168,6 +168,11 @@ For more information, see xref:../installing/installing_aws/installing-restricte
 
 For more information, see the "Installation configuration parameters" section of the installation documentation for your platform.
 
+[NOTE]
+====
+The documentation for disabling the Bare Metal Operator is currently unavailable and is targeted for release at a later date.
```

Comment:
Delete everything after "currently unavailable."

The OpenShift Supplemental Style Guide: https://redhat-documentation.github.io/supplementary-style-guide/#introduction

Only include anticipated releases in the future if absolutely necessary. This doesn't seem to be one of those cases.

Reply:
Done.